packages/node-cli/messages.json

```diff
@@ -122,6 +126,32 @@
 	"COMMAND_LOCAL_RESOURCE_NOT_FOUND": "The process failed. The {0} file was not found in the project.",
 	"COMMAND_LOCAL_INVALID_XML": "Invalid xml {0}",
 
+	"COMMAND_MANAGE_ACCOUNT_ERROR_CREDENTIALS_EMPTY": "There is no authentication ID (authID) configured for this account-role combination. Run \"suitecloud account:setup\" to set up a NetSuite account to use with SuiteCloud CLI for Node.js.",
+	"COMMAND_MANAGE_ACCOUNT_ERROR_TAG": "The Authentication Tag/Label provided, is not valid, please review the data.",
+	"COMMAND_MANAGE_ACCOUNT_ERROR_UNKNOWN_ACTION": "Unknown selected account:manageauth action",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_ACCOUNT_ID": "Account ID: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_ACCOUNT_NAME": "Account Name: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_ACCOUNT_TYPE": "Account Type: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_AUTHID": "Authentication ID (authID): {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_DOMAIN": "Domain: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_ROLE": "Role: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_CANCEL": "The manageauth process has been canceled.",
```

Comment:
I think we should either say "account:manageauth" or just: "The process has been canceled"

Reply:
Hey Viktor, I have consulted it with Alba and she thinks it fine. Also I think that this message is fine because it's consistent with other messages already in the application like "The createproject process has been canceled" and "The setupaccount process has been canceled".
In addition to that, we should think that the name of the process is not exactly the name of the command.